README.md

```diff
@@ -1,17 +1,74 @@
 # Rapid Experiments Shared ![CircleCI](https://img.shields.io/circleci/build/github/mozilla/rapid-experiments-shared)
 
-This is a place to define data and schemas used across the rapid experiments
-program.
+This is a place to define data and schemas used across the rapid experiments program.
 
-Any data that moves between systems should have a schema defined here. Schemas
-are defined using Typescript, and automatically converted to JSON Schema. Any
-data that needs to be re-used by multiple systems should be stored here to be
-shared.
+Any data that moves between systems should have a schema defined here. Schemas are defined using
```

Comment:
You could say `Any data that moves between systems should have typescript types defined here, which will be automatically converted to JSON Schemas`

Reply:
I didn't actually make any changes here, Prettier just re-applied itself with the new 100 char line length. I probably shouldn't have committed this part. (The prettier config is not yet in the repo, I plan to add it later).